src/snapshots/fuzz_crash/fuzz_crash_001.txt

```diff
@@ -1,17 +1,21 @@
 ~~~META
 description=fuzz crash
+verbose-tokens
 ~~~SOURCE
 mo|%
 ~~~PROBLEMS
 PARSER: missing_header
 PARSER: pattern_unexpected_token
 PARSER: unexpected_token
 ~~~TOKENS
-LowerIdent,OpBar,OpPercent,EndOfFile
+@0-2	LowerIdent
```

Comment:
I think these should still be line:col and not byte offset. I think that is still more human understandable in general.

Reply:
Yeah, good idea